deployments/nomad/otel-agent.nomad

```diff
@@ -5,7 +5,7 @@ job "otel-agent" {
   constraint {
     attribute = "${attr.nomad.version}"
     operator  = "semver"
-    value     = "< 1.3.0"
+    value     = "< 1.8.0"
```

Comment:
should this be `>= 1.6.2, , < 1.8.0` instead? Not sure we are checking for version 1.8.0 when the readme say 1.6.2+

Reply:
Sorry, I had removed 1.6.2 from the test matrix. I was ambivalent as to whether we needed to have it on, but I guess we should.